framework/elements/TextBox.js

```diff
@@ -0,0 +1,29 @@
+const BaseElement = require('./BaseElement');
+
+class TextBox extends BaseElement {
+  constructor(name, selector) {
+    super(name, selector);
+  }
+
+  clear() {
+    const element = super.findElement();
+    element.clearValue();
+  }
+
+  typeText(text) {
+    const element = super.findElement();
+    element.setValue(text);
+  }
+
```

Comment:
please add a wrapper fro addValue (which doesn't clear textbox prior to sending a text)